<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_memory">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="RA">
            <property name="text">
             <string>RA</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="PC">
            <property name="text">
             <string>PC</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="RD">
            <property name="text">
             <string>RD</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="RB">
            <property name="text">
             <string>RB</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="RC">
            <property name="text">
             <string>RC</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="SP">
            <property name="text">
             <string>SP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="RAlineEdit"/>
          </item>
          <item row="1" column="6">
           <widget class="QLineEdit" name="SPlineEdit"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="RDlineEdit"/>
          </item>
          <item row="0" column="6">
           <widget class="QLineEdit" name="RClineEdit"/>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="RBlineEdit"/>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="PClineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="stop">
          <property name="text">
           <string>stop</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="help">
              <property name="text">
               <string>help</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sample1">
              <property name="text">
               <string>sample1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sample2">
              <property name="text">
               <string>sample2</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="assemble">
              <property name="text">
               <string>assemble</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="step">
              <property name="text">
               <string>step</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="run">
              <property name="text">
               <string>run</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>main</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_flag"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_stop">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="load">
          <property name="text">
           <string>load</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="save">
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
